test/v6/lossClaims/delete/def1/Def1_DeleteLossClaimValidatorSpec.scala

```diff
@@ -33,45 +33,71 @@ class Def1_DeleteLossClaimValidatorSpec extends UnitSpec {
   private val invalidNino    = "badNino"
   private val validClaimId   = "AAZZ1234567890a"
   private val invalidClaimId = "not-a-claim-id"
+  private val validTaxYear   = "2019-20"
 
   private val parsedNino    = Nino(validNino)
   private val parsedClaimId = ClaimId(validClaimId)
+  private val parsedTaxYear = TaxYear.fromMtd(validTaxYear)
 
-  private def validator(nino: String, claimId: String): Validator[DeleteLossClaimRequestData] = new Def1_DeleteLossClaimValidator(nino, claimId)
+  private def validator(nino: String, claimId: String, taxYear: String): Validator[DeleteLossClaimRequestData] =
+    new Def1_DeleteLossClaimValidator(nino, claimId, taxYear)
 
   "running a validation" should {
     "return the parsed request data" when {
-      "given a valid request" in {
-        val result: Either[ErrorWrapper, DeleteLossClaimRequestData] = validator(validNino, validClaimId).validateAndWrapResult()
+      "passed a valid request" in {
+        val result: Either[ErrorWrapper, DeleteLossClaimRequestData] = validator(validNino, validClaimId, validTaxYear).validateAndWrapResult()
         result shouldBe Right(
-          Def1_DeleteLossClaimRequestData(parsedNino, parsedClaimId)
+          Def1_DeleteLossClaimRequestData(parsedNino, parsedClaimId, parsedTaxYear)
         )
       }
     }
 
     "return NinoFormatError error" when {
-      "given an invalid nino" in {
-        val result = validator(invalidNino, validClaimId).validateAndWrapResult()
+      "passed an invalid nino" in {
+        val result = validator(invalidNino, validClaimId, validTaxYear).validateAndWrapResult()
         result shouldBe Left(
           ErrorWrapper(correlationId, NinoFormatError)
         )
       }
     }
 
-    "return LossIdFormatError error" when {
-      "given an invalid claim ID" in {
-        val result = validator(validNino, invalidClaimId).validateAndWrapResult()
+    "return ClaimIdFormatError error" when {
+      "passed an invalid claim ID" in {
+        val result = validator(validNino, invalidClaimId, validTaxYear).validateAndWrapResult()
         result shouldBe Left(
           ErrorWrapper(correlationId, ClaimIdFormatError)
         )
       }
     }
 
+    "return TaxYearFormatError error" when {
+      "passed an incorrectly formatted taxYear" in {
+        val result = validator(validNino, validClaimId, "202324").validateAndWrapResult()
+
+        result shouldBe Left(ErrorWrapper(correlationId, TaxYearFormatError))
+      }
+    }
+
+    "return a RuleTaxYearNotSupportedError error" when {
+      "passed a taxYear before the minimum supported" in {
```

Comment:
Please update to => passed a **taxYearClaimedFor** before the minimum supported, thanks